### DISCUSSION TITLE
EnC working again; thanks!

### DISCUSSION THREAD

**Discussion by thomiel** [CommunityToolkit/Maui#496] [General]
Just wanted to thank for 1.1.0 in which Edit and Continue does work again. (Issue https://github.com/CommunityToolkit/Maui/issues/402 was prematurely locked and forwarded to the Visual Studio team.)

**Comment by VladislavAntonyuk** [comment]
Nice to hear that. @cargnel could you also confirm that it works?

**Reply by cargnel** [reply]
I confirm

**Comment by thomiel** [comment]
[This change](https://github.com/CommunityToolkit/Maui/commit/c1965ee3d9ad09e98bd948acfa9c6982e96ca557#diff-31dc257ef11d933de8d06f55bc1d053a9d087b6a7228e3d2c561845c7ae97bceL178) lead to constantly changed filenames that maybe were conceived as 'rude change'  by EnC.

**Reply by pictos** [reply]
Oh so I broke the EnC before kkkkkk Glad that I was able fix it (without intention xD)

**Comment by CliffAgius** [comment]
Closed as answered
